detail.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
```

Comment:
are all of these imports necessary? You should remove the ones you aren't using in this file